DEVELOPMENT.md

```diff
@@ -135,12 +135,10 @@ To make changes to these CRDs, you will probably interact with:
 ## Install Pipeline
 
 You can stand up a version of this controller on-cluster (to your
-`kubectl config current-context`), including `knative/build` (which is wrapped
-by [`Task`](docs/Concepts.md#task)):
+`kubectl config current-context`)):
```

Comment:
sorry for the nit, but it has two closing brackets `))`

Reply:
Oh shoot, I'm so sorry for that. 
Thanks a lot for pointing out.